invenio_records_rest/config.py

```diff
@@ -58,20 +56,159 @@ def _(x):
+            'list_route': '/records/',
+            'max_result_window': 10000,
+            'pid_fetcher': '<registered-pid-fetcher>',
+            'pid_minter': '<registered-minter-name>',
+            'pid_type': '<record-pid-type>',
+            'read_permission_factory_imp': permission_check_factory(),
+            'record_class': 'mypackage.api:MyRecord',
+            'record_loaders': {
+                'application/json': 'mypackage.loaders:json_loader'
+            },
+            'record_serializers': {
+                'application/json': 'mypackage.utils:my_json_serializer'
+            },
+            'search_class': 'mypackage.utils:mysearchclass',
+            'search_factory_imp': search_factory(),
+            'search_index': 'elasticsearch-index-name',
+            'search_serializers': {
+                'application/json': 'mypackage.utils:my_json_search_serializer'
+            },
+            'search_type': 'elasticsearch-doc-type',
+            'suggesters': {
+                'my_url_param_to_complete': {
+                    'completion': {
+                        'field': 'suggest_byyear_elasticsearch_field',
+                        'size': 10,
+                        'context': 'year'
+                    }
+                },
+            },
+            'update_permission_factory_imp': permission_check_factory(),
+            'use_options_view': True,
+        },
+    }
+
+:param create_permission_factory_imp: Import path to factory that crcreate
+    permission object for a given record.
+
+:param default_media_type: Default media type for both records and search.
+
+:param delete_permission_factory_imp: Import path to factory that creates a
+    delete permission object for a given record.
+
+:param item_route: URL template for a single record.
+
+:param links_factory_imp: Factory for record links generation.
+
+:param list_route: Base URL for the records endpoint.
+
+:param max_result_window: Maximum total number of records retrieved from a
+    query.
+
+:param pid_type: It specifies the record pid type. It's used also to build the
+    endpoint name. Required.
+    E.g. ``url_for('record-pid-type_list')`` to point to the records list.
+
+:param pid_fetcher: It identifies the registered fetcher name. Required.
+
+:param pid_minter: It identifies the registered minter name. Required.
+
+:param read_permission_factory_imp: Import path to factory that creates a
+    read permission object for a given record.
+
+:param record_class: Name of the record API class.
+
+:param record_loaders: It contains the list of record deserializers for
+    supperted formats.
+
+:param record_serializers: It contains the list of record serializers for
+    supported formats.
+
+:param search_class: Import path or class object for the object in charge of
+    execute the search queries. The default search class is
+    :class:`invenio_search.api.RecordsSearch`.
+    For more information about resource loading, see the Search of
```

Comment:
I had some issues to point directly.
I think because they are not exporting docs from the classes but writing inside `docs/` everything.
Do you have this issue?